pallets/utxo/src/tests.rs

```diff
@@ -601,3 +602,96 @@ fn test_send_to_address() {
         );
     })
 }
+
+// Generate block time in seconds
+fn gen_block_time_real() -> impl Strategy<Value = RawBlockTime> {
+    (RawBlockTime::THRESHOLD..RawBlockTime::THRESHOLD + 1_000_000_000).prop_map(RawBlockTime::new)
```

Comment:
What is this 1_000_000_000?

Reply:
That is just an arbitrary upper bound for generated timestamps. Can change it to `2 * THRESHOLD`, that may be more clear.

Reply:
Just an arbitrary upper bound fr generated timestamps. The result is `3 * THRESHOLD` so can change it to that for more clarity.

Reply:
Either that or just a comment. I'm fine with either.